app/cni/pkg/install/utils.go

```diff
@@ -28,17 +28,22 @@ func parseBytesToHashMap(bytes []byte) (map[string]interface{}, error) {
 	return parsed, nil
 }
 
-func CreateNewLogger(name string, logLevel kuma_log.LogLevel) logr.Logger {
+func CreateNewLogger(name string, logLevel kuma_log.LogLevel, logFormat kuma_log.LogFormat) logr.Logger {
 	// kubelet expects a specific JSON on stdout, so we're using stderr in CNI
-	return core.NewLoggerTo(os.Stderr, logLevel).WithName(name)
+	return core.NewLoggerTo(os.Stderr, logLevel, logFormat).WithName(name)
 }
 
-func SetLogLevel(logger *logr.Logger, level string, name string) error {
-	logLevel, err := kuma_log.ParseLogLevel(level)
-	if err != nil {
-		return err
+func SetLogUtils(logger *logr.Logger, level string, name string, format string) error {
```

Comment:
Are level, name and format utils?

How about we call it `ConfigureLogger`

Reply:
Yes that would be better.